fenwick_tree/cpp/fenwick_tree.cpp

```diff
@@ -0,0 +1,52 @@
+#include <iostream>
+#include <vector>
+using namespace std;
+struct FenwickTree {
+	vector<int> v;
+	int length;
+	FenwickTree(int l) : length(l) {
+		v.assign(length + 1, 0);
+	}
```

Comment:
Single-parameter constructors should be marked explicit.  [runtime/explicit] [5]

*Origin: CPPLintBear, Section: `all.cpplint`.*